pkg/piecestore/rpc/client/utils/utils.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package utils
```

Comment:
Does this function deserve a separate package? Can't it be part of the rpc/client package, yet in a separate file?

Reply:
@aleitner